### PR TITLE
Adds Ollama Grid Search to Community integrations on README

### PR DESCRIPTION
**Pull request by dezoito**
Adds the following content to the  Community Integrations section:

### Model/Prompt Evaluation and Optimization

- [Ollama Grid Search](https://github.com/dezoito/ollama-grid-search) (Multi-platform desktop application)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,6 +300,10 @@ See the [API documentation](./docs/api.md) for all endpoints.
 - [BrainSoup](https://www.nurgo-software.com/products/brainsoup) (Flexible native client with RAG & multi-agent automation)
 - [macai](https://github.com/Renset/macai) (macOS client for Ollama, ChatGPT, and other compatible API back-ends)
 
+### Model/Prompt Evaluation and Optimization
```
Comment (jmorganca) [inline README.md:303]:
```suggestion
### Evaluation
```

Comment (jmorganca) [reply README.md:303]: This is a great new section to start

**Review by dezoito — COMMENTED** [review]
Looks good to me!

**Comment by dezoito** [comment]
@jmorganca, since your changes were accepted, can this be merged?

TY!

**Review by mchiang0610 — APPROVED** [review]

**Pull request merged**